src/document/position.js

```diff
@@ -97,13 +119,13 @@ CKEDITOR.define( [ 'utils', 'ckeditorerror' ], ( utils, CKEditorError ) => {
 		}
 
 		/**
-		 * Two positions equal if paths are equal.
+		 * Two positions equal if paths are equal and roots are the same.
 		 *
 		 * @param {document.Position} otherPosition Position to compare.
 		 * @returns {Boolean} True if positions equal.
 		 */
 		isEqual( otherPosition ) {
```

Comment:
I am not sure if if we need `Position.isEqual` anymore is one can use `Position.compareWith == Position.SAME`. Or if we assume that comparing position will be popular we could add also `Position.isBefore` and `Position.isAfter`.